pkg/uibackend/rest/dashboard_riskiest_assets.go

```diff
@@ -212,13 +212,31 @@ func getAssetInfo(asset *backendmodels.AssetType) (*models.AssetInfo, error) {
 	switch info := discriminator.(type) {
 	case backendmodels.VMInfo:
 		return vmInfoToAssetInfo(info)
+	case backendmodels.ContainerInfo:
+		return containerInfoToAssetInfo(info)
+	case backendmodels.ContainerImageInfo:
+		return containerImageInfoToAssetInfo(info)
 	default:
 		return nil, fmt.Errorf("asset type is not supported (%T)", discriminator)
 	}
 }
 
+func containerInfoToAssetInfo(info backendmodels.ContainerInfo) (*models.AssetInfo, error) {
+	return &models.AssetInfo{
+		Name: info.ContainerName,
+		Type: utils.PointerTo(models.Container),
+	}, nil
+}
+
+func containerImageInfoToAssetInfo(info backendmodels.ContainerImageInfo) (*models.AssetInfo, error) {
+	return &models.AssetInfo{
+		Name: info.Name,
+		Type: utils.PointerTo(models.Container),
```

Comment:
`models.ContainerImage`?
Also, what about `location` here and in `containerInfoToAssetInfo`?

Reply:
I forgot to go back and update these, great catch! I've pushed a new revision.